client/allocdir/alloc_dir.go

```diff
@@ -24,6 +24,10 @@ const (
 )
 
 var (
+	// SnapshotErrorTime is the sentinel time that will be used on the
+	// error file written by Snapshot when it encounters as error.
+	SnapshotErrorTime = time.Date(2000, 0, 0, 0, 0, 0, 0, time.UTC)
```

Comment:
Probably should have a `IsSnapshotErrorFile` method that checks this as well as the file name. Right now this is unused on the validation path